style.css

```diff
@@ -432,3 +432,269 @@ footer {
   justify-content: center;
   align-content: space-between;
 }
+
+@media all and (min-width: 992px) {
+  body {
+    background: #191765;
+    color: #fff;
+  }
+
+  header {
+    margin-top: 0;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .hcontent {
+    display: flex;
+    justify-content: center;
+  }
+
+  .desktop {
+    display: inline-block;
+    margin-right: 10px;
+    color: #fff;
+  }
+
+  .Icons {
+    display: none;
+  }
+
+  .intro {
+    display: grid;
+    gap: 10px;
+  }
+
+  .intro h1 {
+    position: static;
+    font-family: Roboto, Arial, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 90px;
+    color: #9391ff;
+  }
+
+  .intro h1::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    width: 0;
+    height: 184px;
+    border-radius: 50%;
+    background-color: #edecff;
+    box-shadow: 0 0 95px 0 #edecff;
+    filter: blur(10px);
+    margin-left: 50px;
+  }
+
+  .intro h2 {
+    font-family: Roboto, Arial, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 40px;
+    line-height: 47px;
+    text-align: center;
+    color: #dfe1e6;
+    margin-top: -50px;
+  }
+
+  .intro p {
+    margin-right: 30%;
+    margin-left: 30%;
+    color: #b3bac5;
+  }
+
+  .firsthr {
+    margin-left: 30%;
+    margin-right: 30%;
+  }
+
+  .intro p::before {
+    content: '';
+    position: absolute;
+    right: 0;
+    width: 300px;
+    height: 300px;
+    border-radius: 100%;
+    background-color: #fff;
+    box-shadow: 0 0 95px 0 #d1cfff;
+    filter: blur(10px);
+  }
+
+  .abilities {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 25px;
+    margin-left: 30%;
+    margin-right: 20%;
+  }
+
+  .Framworks::before {
+    display: none;
+  }
+
+  .title {
+    font-family: Roboto, Arial, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 18px;
+    color: #9390ff;
+  }
+
+  .mywork1 {
+    display: block;
+  }
+
+  .social-media-myWork {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 106px 142px 237px 142px;
+  }
+
+  .projects {
+    margin-left: 142px;
+    margin-right: 142px;
+  }
```

Comment:
## OPTIONAL 

- [ ] Please align images in this section to look like the original design  in this section 👍🏿 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yepa](https://user-images.githubusercontent.com/69638013/155575723-80a6c30b-e65a-42bf-a6e5-8119273f7877.PNG) | ![gggggggggg](https://user-images.githubusercontent.com/69638013/155575693-bc491030-c7eb-45e3-9513-00a5cf1c7f8b.PNG)